Formula/uutils-findutils.rb

```diff
@@ -0,0 +1,38 @@
+class UutilsFindutils < Formula
+  desc "Cross-platform Rust rewrite of the GNU findutils"
+  homepage "https://github.com/uutils/findutils"
+  url "https://github.com/uutils/findutils/archive/refs/tags/0.3.0.tar.gz"
+  sha256 "0ea77daf31b9740cfecb06a9dbd06fcd50bc0ba55592a12b9f9b74f3302f5c41"
+  license "MIT"
+  head "https://github.com/uutils/findutils.git", branch: "main"
+
+  depends_on "rust" => :build
+  uses_from_macos "llvm"
+
+  def install
+    ENV["LIBCLANG_PATH"] = Formula["llvm"].opt_lib.to_s if OS.linux?
+    system "cargo", "install", *std_cargo_args
+    mkdir_p libexec
+    mv bin, libexec/"uubin"
+    %w[
+      find xargs
+    ].each do |cmd|
```

Comment:
I wonder if we can make this a bit lower-maintenance by checking what's in `/bin` or `/usr/bin` instead of filling this array up manually. See the `coreutils` formula for how this might work.

Reply:
This isn't likely to help here because there's a binary I'm deliberately avoiding (`testing-commandline`) that morally isn't part of the findutils but is used as a testbed in the project.  It gets installed in (what becomes) `uubin` but I don't want to link it (back) to `bin`.

Reply:
Yup, I've seen that. That's easy enough to exclude from linking into `bin`.